addons/im_livechat_mail_bot/controllers/thread.py

```diff
@@ -0,0 +1,12 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.http import request, route
+from odoo.addons.mail.controllers import thread
+
+
+class ThreadController(thread.ThreadController):
+    @route()
+    def mail_message_post(self, thread_model, thread_id, post_data, context=None):
+        if post_data.get("canned_response_ids"):
```

Comment:
If the issue is since 16.4 this should be fixed in 16.4 instead.

Reply:
Done in odoo/odoo#162155 and odoo/odoo#162234